bridge_adaptivity/module/tests/test_views.py

```diff
@@ -184,6 +204,32 @@ def test_update_cg(self):
         self.assertNotEqual(test_g.description, self.test_cg.description)
         self.assertNotEqual(test_g.collections.all(), self.test_cg.collections.all())
 
+    def test_add_group_to_course(self):
+        group = self.create_group()
```

Comment:
I would move this functions create_group and create_course to the setup class instead of using as separate functions.